Samples/DynamicDependencies/README.md

```diff
@@ -0,0 +1,28 @@
+# Use the Windows App SDK from non-MSIX deployed applications
```

Comment:
@btueffers can you ensure the right header gets added here for Samples browser integration? 

Here is an example of the header metadata to add: https://github.com/microsoft/WindowsAppSDK-Samples/blob/unpackaged/Samples/ResourceManagement/README.md

Reply:
```suggestion
---
name: "Dynamic Dependencies Sample"
description: "Demonstrate how to use Dynamic Dependencies API to load MSIX framework packages in unpackaged apps."
page_type: sample
languages:
- csharp
- cpp
products:
- windows
- windows-app-sdk
urlFragment: DynamicDependencies
---

# Use the Windows App SDK from non-MSIX deployed applications
```